getting-started/get-the-sdk.md

```diff
@@ -387,11 +401,83 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
 {% endtab %}
 
 {% tab title="React Native" %}
-### Javascript
 
 For React Native apps, initialize the SDK using native code in your `AppDelegate` and `MainApplication` in iOS and Android, respectively.
 
-The initialization code is located in the [React Native ACPCore Github README](https://github.com/adobe/react-native-acpcore).
+### iOS
+
+```objectivec
+#import "ACPCore.h"
+#import "ACPUserProfile.h"
+#import "ACPIdentity.h"
+#import "ACPLifecycle.h"
+#import "ACPSignal.h"
+...
+@implementation AppDelegate
+-(BOOL)application:(UIApplication *)application didFinishLaunchingWithOptions:(NSDictionary *)launchOptions {
+    [ACPCore setLogLevel:ACPMobileLogLevelDebug];
+    [ACPCore configureWithAppId:@"<your_environment_id_from_Launch>"];
+    [ACPUserProfile registerExtension];
```

Comment:
Do we want to include userprofile in core example?

Reply:
https://aep-sdks.gitbook.io/docs/getting-started/get-the-sdk#2-add-initialization-code
`
The following code snippets demonstrate how you can import and register the Mobile Core and Profile extensions. You can also see, for reference, how Identity, Lifecycle, Signal, Profile, and other extensions are imported and registered.
`
yes, we'd better keep it as we should also show the native code to register other extension.

Reply:
ok